freshmaker/image.py

```diff
@@ -1371,6 +1380,7 @@ def find_images_to_rebuild(
             published,
             release_categories,
             leaf_container_images=leaf_container_images,
+            restrict_to_ext_repos=restrict_to_ext_repos,
```

Comment:
As above, "external repos" is misleading, I'd suggest just use `repositories` here in parameters, it should not be specific to the "compliance priority" case, it can be used in other cases as well, just name `repositories` makes more sense.

Reply:
I liked your suggestion a lot! These parameter names are indeed much more clear.